www/source/partials/docs/_built-in-helpers.html.md.erb

```diff
@@ -79,4 +79,26 @@ like this:
 
     [[servers]]
     host = "host-2"
-    port = 3434
\ No newline at end of file
+    port = 3434
+
+And for both `each` and `unless`, you can use `@first` and `@last` to specify which item in an array you want to perform business logic on. For example:
+
+    "mongo": {
+      {{~#each bind.database.members as |member|}}
```

Comment:
I think in core plans were trying to standardize around `~` at the end of the handlebars syntax. https://github.com/habitat-sh/core-plans/pull/986/files